code/datums/components/uplink.dm

```diff
@@ -181,6 +181,16 @@
 							break
 					if(is_inaccessible)
 						continue
+			if(I.restricted_traits && I.discounted == FALSE)
+				if(ishuman(user))
+					var/is_inaccessible = TRUE
+					var/mob/living/carbon/human/H = user
+					for(var/T in I.restricted_traits)
```

Comment:
use `find` x `in` y instead.

Reply:
how? I cannot find any proc called find anywhere?

Reply:
my bad, it's `locate`

Reply:
I couldn't figure out how to make locate work without breaking the entire uplink, I got it to compile but it behaved oddly :/